sdk/eventhub/azure-eventhub/stress/templates/testjob.yaml

```diff
@@ -9,8 +9,52 @@ spec:
   containers:
     - name: python-eh-stress
       image: {{ .Values.image }}
-      command: ['bash', '-c', 'python3 azure_eventhub_producer_stress.py & python3 azure_eventhub_consumer_stress_sync.py']
+      imagePullPolicy: Always
+     
+      {{ if eq .Stress.Scenario "identity" }}
+      command: ['bash', '-c', 'python azure_eventhub_producer_stress.py -m stress_send_sync --azure_identity True --pyamqp_logging_enable --print_console --duration 7200']
+      {{- end -}}
+
+      {{ if eq .Stress.Scenario "sendsync" }}
+      command: ['bash', '-c', 'python azure_eventhub_producer_stress.py -m stress_send_sync --duration 7200']
```

Comment:
FYI @ckairen a scenario to consider testing your matrix changes with, e.g. coming up with good examples of custom commands per scenario.